packages/web-components/src/components/quote/index.ts

```diff
@@ -12,3 +12,4 @@ import './quote-source-bottom-copy';
 import './quote-source-copy';
 import './quote-source-heading';
 import '../link-with-icon/link-with-icon';
```

Comment:
Is this element import still needed here since `quote-link-with-icon` is extending it?

Reply:
Ahh yeah quite right. For a second I thought we still needed it for backward compatibility, but we still get it by way ot `quote-link-with-icon`. Will fix.